scripts/populate_store/upload_bundles.py

```diff
@@ -0,0 +1,38 @@
+import sys
+import json
+import requests
+
+
+def upload_structure_definitions(bundle_files):
+
+    resource_types = ["StructureDefinition", "ConceptMap", "CodeSystem"]
+    api = "https://fhir.staging.arkhn.org/api"
+
+    for bundle_file in bundle_files:
+        if bundle_file.endswith(".json"):
+            with open(bundle_file) as f:
+                bundle = json.load(f)
+
+            assert (
+                "resourceType" in bundle and bundle["resourceType"] == "Bundle"
+            ), f"{bundle_file} must be a FHIR Bundle resource"
+
+            for entry in bundle["entry"]:
+                if "resource" not in entry:
+                    raise Exception("Bundle entry is missing a resource.")
+
+                if entry["resource"]["resourceType"] not in resource_types:
+                    continue
+
+                resource_type = entry["resource"]["resourceType"]
+                response = requests.post(f"{api}/{resource_type}", json=entry["resource"])
+                if response.status_code != 200:
+                    print(response.text)
```

Comment:
can you use the logging library (with .error) instead ? and have a nice format as well